include/variables.h

```diff
@@ -217,6 +217,7 @@ extern volatile OSTime gRSPGfxTimeAcc;
 extern volatile OSTime gRSPOtherTimeAcc;
 extern volatile OSTime D_8016A578;
 extern volatile OSTime gRDPTimeAcc;
+extern struct SpeedMeterTimeEntry* sSpeedMeterTimeEntryPtr; // declared here purely for bss ordering reasons
```

Comment:
is this because fix_bss.py is bugged, or because there's no way to make `increment_block_number` work?